tools/ingest/retrieve_data_from_noaa_nomads.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+##################################################################
+# 
+#  PURPOSE:
+#       retrieve BUFR data from NOAA NOMADS server
+#         optionally provide a data type
+#         will default to AMSU-A, MHS, GNSS-RO and IASI
+#
+#      NOTE: NOMADS keeps ~10days of data online
+#
+#   REQUIREMENTS:
+#       wget
+#
+##################################################################
+
+nomads_address='https://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/prod'
```

Comment:
`gfs` == early cycle and `gdas` == late cycle. Suggest either 1) changing to `gdas` everywhere or 2) making `dump` another option for the user.

Reply:
excellent,,, I never knew about this

could have it try `gdas` first and only try `gfs` if no data is found?

Reply:
So I misspoke. This URL should be `gfs` regardless, but the below in `get_files` can be either `gfs` or `gdas`.

I think for super near real time, `gfs` will exist before `gdas` (there is about a 3 hour difference between the two). In terms of observations, `gdas` will have everything in `gfs` plus all late arriving obs. So I would say yes, have it try `gdas` first and then fail to `gfs`.

Reply:
For example: `https://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/prod/gdas.20211210/12/atmos/` vs `https://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/prod/gfs.20211210/12/atmos/`